app/renuo_upload_signing.rb

```diff
@@ -44,6 +45,18 @@ class RenuoUploadSigning < Sinatra::Base
     body 'up'
   end
 
+  delete '/delete_file' do
+    set_response_headers
+    content_type :json
+    api_key = settings.authentication.api_key_or_nil(params[:api_key])
+    if settings.authentication.private_api_key_valid?(api_key, params[:private_api_key])
+      status 200
+      settings.s3_service.delete_file(api_key.full_app_name, params[:file_path])
```

Comment:
we can't. there is no way to find out, since AWS delete is silent. you won't get a different status code or response if the request was not successful.